services/draft/driven/rdb/src/main/java/nettee/draft/driven/rdb/persistence/mapper/DraftEntityMapper.java

```diff
@@ -13,11 +15,14 @@
 
 @Mapper(componentModel = "spring")
 public interface DraftEntityMapper {
+
     Draft toDomain(DraftEntity draftEntity);
     DraftEntity toEntity(Draft draft);
     DraftDetail toDraftDetail(DraftEntity draftEntity);
     DraftSummary toDraftSummary(DraftEntity draftEntity);
     DraftTitle toDraftTitle(DraftEntity draftEntity);
+    DraftImage toDomain(DraftImageEntity draftImageEntity);
+    DraftImageEntity toEntity(DraftImage draftImage);
```

Comment:
이 위에 공백줄을 추가하면 어떨까요?
어느 정도 섹션 구분이 있으면 보기 편할 듯합니다!

```suggestion

    DraftImage toDomain(DraftImageEntity draftImageEntity);
    DraftImageEntity toEntity(DraftImage draftImage);
```

Reply:
빠른 리뷰 감사드립니다!
수정 및 병합 완료하였습니다!!!!